### DISCUSSION TITLE
Use scanner devices

### DISCUSSION THREAD

**Discussion by fssherwani** [mebjas/html5-qrcode#328] [Q&A]
Hello, will this library work with scanning devices ?

**Comment by ROBERT-MCDOWELL** [comment]
if it's infrared i doubt it.

**Reply by fssherwani** [reply]
Ok thanks for reply, btw is there any JavaScript Qr code library for infrared device ?

**Comment by ROBERT-MCDOWELL** [comment]
you should search the device manufacturer website that usually provide the drivers and how it is managed from javascript.

**Comment by mebjas** [comment]
Can you share what kind of infrared scanner is used?

It depends if the device reads it as a camera device (which I suspect).
